### ISSUE TITLE
[Feature Request]: Show a message instead of hiding the community tab on the subscriptions page when it is unavailable because of RSS

### ISSUE THREAD

**Issue by riverpiers** [FreeTubeApp/FreeTube#5716]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

When the user has 125 or more subscriptions in the currently active profile FreeTube switches to RSS (as indicated by a toast message displayed at the time) and as the community posts are not available over RSS FreeTube currently hides that tab. To users it is not directly obivious on the subscriptions page itself why the community tab was hidden.

### Proposed Solution

Instead of hiding the community tab when using RSS, FreeTube should instead display a message inside of the tab explaining why community posts cannot be shown.

### Issue Labels

ease of use improvement, new feature, new optional setting

**Comment by absidue** [comment]
That is not a bug, when you have 125 channels or more in a profile, FreeTube switches to the RSS feeds (it shows a message telling you about it), so avoid getting ratelimited by YouTube and YouTube doesn't have any RSS feeds for the community posts.

**Comment by riverpiers** [comment]
Yes, it automatically switches to RSS on all channels because I have 300+ subscriptions and there is no community tab here. Only when I switch to segregated/thematic subscription categories did I notice that above 125, the community tab disappears. It is not obvious when sorting subscriptions by separate categories/topics and there are fewer than 125 subscriptions. Maybe some info in the app about when to expect the deactivation of the community tab? That’s all from my side, do as you see fit. It just wasn’t obvious since the tab wasn’t visible when it immediately switched to RSS, and in selected/segregated subscriptions as a separate category/topic, the tab appears.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> Instead of hiding the community tab when using RSS, FreeTube should instead display a message inside of the tab explaining why community posts cannot be shown.

The message should at least include the reason why its empty (amount of subscriptions, 125) and path to the setting that hides this tab
